src/tests/integration/test_generic_db_opeations.py

```diff
@@ -1,22 +1,35 @@
 import pytest
+
 # flake8: noqa
 from fixtures.database import *
```

Comment:
star imports are discouraged, either do `from fixtures import database` or list them one by one

Reply:
check out pytest docs, you can define a `conftest.py` file with all your fixtures, if you have it, then you don't need to import the fixtures, which I believe is what you want to do, since pytest will import them automatically. I think you can you `tests/integration/conftest.py` and put your tests under `tests/integration/` this way the conftest will be local to the integration tests

Reply:
Gotcha, that's what I want

Fix in [415a581](https://github.com/ploomber/jupysql/pull/117/commits/415a581b29f8a50bb976efd4ea55efafc147b65a)